docker-rsync/hosts-init

```diff
@@ -2,5 +2,8 @@
 
 set -eu
 
-touch $HOME/.ssh/known_hosts
+if [ ! -f "$HOME/.ssh/known_hosts" ]; then
+    touch $HOME/.ssh/known_hosts
```

Comment:
`touch -f $HOME...` would also be an acceptable alternative, if you wanted to inline it.